server/models/post.js

```diff
@@ -0,0 +1,62 @@
+const { init } = require('../dbConfig');
+const { ObjectId } = require('mongodb');
+
+
```

Comment:
.gitignore is buggy sometimes but make sure you get it actually working before pushing node_modules. As there are 500+ files